src/Controller/Rest/ApiController.php

```diff
@@ -51,14 +61,16 @@ public function getApi(Api $api): Response
 
     /**
      * @param Request $request
+     * @param TokenInterface $token
      * @return Response
      *
-     * @Route("/", methods={"POST"})
+     * @Route("/", methods={"POST", "GET"})
      */
     public function createApi(Request $request): Response
     {
-        $api = $this->apiService->createApi($request->get('name'), $request->get('description'), $this->getUser());
-        return new Response($this->serialize($api), Response::HTTP_CREATED);
+        header("Access-Control-Allow-Origin: *");
```

Comment:
Useless

Reply:
been here by chance "why useless tho"

Reply:
Sorry mate I only see this now and wouldn't be able to help you, I don't remember at all and haven't touched symfony/php since 2019